tikv/kv.go

```diff
@@ -582,6 +583,26 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
 	wg.Wait()
 }
 
+// EnableResourceControl enables the resource control.
+func (s *KVStore) EnableResourceControl() {
```

Comment:
I think no need be a method of KVStore, directly as ` func EnableResourceControl() `.